### ISSUE TITLE
Extraneous semicolons from stacked inline class definitions

### ISSUE THREAD

**Issue by mjago** [ruby-formatter/rufo#59]
```echo "class A end; class B end;class C end # comment" | rufo```
gives:
```ruby
class A end; 
class B end; 
class C end # comment
```

**Comment by mjago** [comment]
The above code is a syntax error in v2.2.8. However the following
```echo "class A; end; class B; end;class C; end # comment" | rufo```
gives:
```ruby
class A; end; 
class B; end; 
class C; end # comment
```

**Comment by mjago** [comment]
Fixed with #60
